src/components/__tests__/AppNavigation.test.js

```diff
@@ -119,24 +119,54 @@ describe('AppNavigation.vue', () => {
     expect(wrapper.vm.getCurrentPageIndex()).toBe(2)
   })
 
-  it('should display number of new messages', () => {
+  it('badge should be hidden when there are no messages', () => {
```

Comment:
"it badge should be..." is incorrect. Test is describing 'AppNavigation.vue', so you should write tests on its behalf:

```js
it('should hide the badge when there are no messages', /* ... */)
```
and it should be like that everywhere

Reply:
The "badge" is part of the AppNavigation component, so it should be tested within this component.

Reply:
I mean the test block is describing 'AppNavigation.vue':
```js
describe('AppNavigation.vue', () => {
```
and then you write "**it** badge should", which is not a right English sentence:
```js
it('badge ...'
```
It should be "it**s** badge", since "it" refers to 'AppNavigation.vue', but you can write "It (_AppNavigation.vue_) should hide the badge...":
```js
it('should hide the badge...'
```
The point is that the name of the method is part of the sentence. You can also just use `test` instead of `it`